addon/mixins/new-popover-modal-api.js

```diff
@@ -0,0 +1,30 @@
+export default Ember.Mixin.create({
+  init(...args) {
+    this._super(...args);
+    let opts = this.get('_propertyOptions');
+    let isNewApi = this.get('__renderPopoverNewAPI');
+    Ember.assert(`[ember-widgets] popover or modal must be initialized with _propertyOptions`, !isNewApi || !!opts);
+    this._setupProperties(opts);
+  },
+  _setupProperties(props={}) {
+    let boundPropKeys = [];
+    let bareProps = {};
+    let BINDING_RE = /Binding$/;
```

Comment:
Any reason to keep this a local variable over an actual `const` at the top of the file?

Reply:
Could we perhaps add a comment explaining "binding" props vs "bare" props? I'm not entirely clear on what these are being aliased to/why the aliases are necessary.